### PR TITLE
add modelAPI to pedestian_tracker_demo

### PR DESCRIPTION
**Pull request by VoronovaIntern**
redesign demo to OMZ Model  API approach

- [x] apply Model API wrappers for detection models
- [ ] apply Model API wrappers for reidentification models (this postponed until model api architecture discussion)

### CODE REVIEW

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 1]
```diff
@@ -65,9 +112,9 @@ CreatePedestrianTracker(const std::string& reid_model,
         tracker->set_descriptor_strong(descriptor_strong);
         tracker->set_distance_strong(distance_strong);
     } else {
-        slog::warn << "Reid model "
+        std::cout << "WARNING: Reid model "
             << "was not specified. "
-            << "Only fast reidentification approach will be used." << slog::endl;
+            << "Only fast reidentification approach will be used." << std::endl;
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:117]: Please, undo these changes. For logging you should use `slog `.

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 2]
```diff
@@ -127,13 +182,47 @@ int main(int argc, char **argv) {
...
+        }
+        else {
+            slog::err << "No model type or invalid model type (-at) provided: " + FLAGS_at << slog::endl;
+            return -1;
+        }
+        
+      
+
         std::vector<std::string> devices{detector_mode, reid_mode};
-        slog::info << *GetInferenceEngineVersion() << slog::endl;
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:131]: Need to revert.

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 3]
```diff
@@ -160,13 +249,88 @@ int main(int argc, char **argv) {
         cv::Size graphSize{static_cast<int>(frame.cols / 4), 60};
         Presenter presenter(FLAGS_u, 10, graphSize);
 
+        std::cout << "To close the application, press 'CTRL+C' here";
+        if (!FLAGS_no_show) {
+            std::cout << " or switch to the output window and press ESC key";
+        }
+        std::cout << std::endl;
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:256]: Remove this message, it's unnecessary.

**Comment by ivikhrev** [comment]
Pls, rebase to the latest develop

**Comment by ivikhrev** [comment]
Jenkins ok to test

File: demos/tests/cases.py [hunk 4]
```diff
@@ -353,9 +353,11 @@ def single_option_cases(key, *args):
         TestCase(options={'-no_show': None,
             **MONITORS,
             '-i': DataPatternArg('person-detection-retail')}),
+        *combine_cases(
+                TestCase(options={'-at': 'ssd'}),
         single_option_cases('-m_det',
             ModelArg('person-detection-retail-0002'),
-            ModelArg('person-detection-retail-0013')),
+            ModelArg('person-detection-retail-0013'))),
```
Comment (vladimir-dudnik) [inline demos/tests/cases.py:360]: it seems models.lst and test cases should be extended to cover new supported models

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 5]
```diff
@@ -126,14 +143,46 @@ int main(int argc, char **argv) {
...
+        else if (FLAGS_at == "ssd") {
+            detection_model.reset(new ModelSSD(det_model, (float)FLAGS_t, FLAGS_auto_resize, labels));
+        }
+        else if (FLAGS_at == "yolo") {
+            detection_model.reset(new ModelYolo(det_model, (float)FLAGS_t, FLAGS_auto_resize, FLAGS_yolo_af, (float)FLAGS_iou_t, labels));
+        }
+        else {
+            slog::err << "No model type or invalid model type (-at) provided: " + FLAGS_at << slog::endl;
+            return -1;
+        }
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:173]: Part of this networks don't detect persons (faceboxes, retinaface, retinaface-pytorch), you should remove them, and for those that detect filter boxes with ID corresponds to persons (centernet, ssd, yolo networks).

File: demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp [hunk 6]
```diff
@@ -40,6 +40,18 @@ static const char delay_message[] = "Optional. Delay between frames used for vis
 static const char output_log_message[] = "Optional. The file name to write output log file with results of pedestrian tracking. "
                                           "The format of the log file is compatible with MOTChallenge format.";
 static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+//added
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp:43]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp [hunk 7]
```diff
@@ -56,6 +68,16 @@ DEFINE_bool(no_show, false, no_show_message);
 DEFINE_int32(delay, 3, delay_message);
 DEFINE_string(out, "", output_log_message);
 DEFINE_string(u, "", utilization_monitors_message);
+//added
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp:71]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 8]
```diff
@@ -94,6 +105,12 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
         throw std::logic_error("Parameter -m_reid is not set");
     }
 
+    //add FLAGS_at from model API
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:108]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 9]
```diff
@@ -160,10 +209,72 @@ int main(int argc, char **argv) {
         Presenter presenter(FLAGS_u, 10, graphSize);
 
         for (unsigned frameIdx = 0; ; ++frameIdx) {
-            pedestrian_detector.submitFrame(frame, frameIdx);
-            pedestrian_detector.waitAndFetchResults();
+            //model API part
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:212]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 10]
```diff
@@ -160,10 +209,72 @@ int main(int argc, char **argv) {
...
+                {
+                    res.outputsData.emplace(outName, std::make_shared<TBlob<int>>(*as<TBlob<int>>(blobPtr)));
+
+                }
+
+                else
+                {
+                    res.outputsData.emplace(outName, std::make_shared<TBlob<float>>(*as<TBlob<float>>(blobPtr)));
+
+                }
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:239]:
```suggestion
                if (Precision::I32 == blobPtr->getTensorDesc().getPrecision()) {
                    res.outputsData.emplace(outName, std::make_shared<TBlob<int>>(*as<TBlob<int>>(blobPtr)));
                }
                else {
                    res.outputsData.emplace(outName, std::make_shared<TBlob<float>>(*as<TBlob<float>>(blobPtr)));
                }
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 11]
```diff
@@ -160,10 +209,72 @@ int main(int argc, char **argv) {
...
+            }
+
+
+            auto result = (detection_model->postprocess(res))->asRef<DetectionResult>();
+
+            TrackedObjects detections;
+
+
+            for (size_t i = 0; i < result.objects.size(); i++)
+            {
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:249]:
```suggestion
            for (size_t i = 0; i < result.objects.size(); i++) {
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 12]
```diff
@@ -160,10 +209,72 @@ int main(int argc, char **argv) {
...
+                object.rect = cv::Rect2f(cv::Point(static_cast<int>(round(static_cast<double>(x0))),
+                    static_cast<int>(round(static_cast<double>(y0)))),
+                    cv::Point(static_cast<int>(round(static_cast<double>(x1))),
+                        static_cast<int>(round(static_cast<double>(y1)))));
+
+                if (object.rect.area() > 0) {
+                    detections.emplace_back(object);
+                }
+
+
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:275]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 13]
```diff
@@ -126,14 +143,46 @@ int main(int argc, char **argv) {
...
         bool should_save_det_log = !detlog_out.empty();
 
+        //model API part
+        std::vector<std::string> labels;
+        if (!FLAGS_labels.empty())
+            labels = DetectionModel::loadLabels(FLAGS_labels);
+
+        std::unique_ptr<ModelBase> detection_model;
+        if (FLAGS_at == "centernet") {
+            detection_model.reset(new ModelCenterNet(det_model, (float)FLAGS_t, labels));
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:153]: Since you are refactoring this demo, pls allign code style to camelCase

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 14]
```diff
@@ -126,14 +143,46 @@ int main(int argc, char **argv) {
...
+            detection_model.reset(new ModelSSD(det_model, (float)FLAGS_t, FLAGS_auto_resize, labels));
+        }
+        else if (FLAGS_at == "yolo") {
+            detection_model.reset(new ModelYolo(det_model, (float)FLAGS_t, FLAGS_auto_resize, FLAGS_yolo_af, (float)FLAGS_iou_t, labels));
+        }
+        else {
+            slog::err << "No model type or invalid model type (-at) provided: " + FLAGS_at << slog::endl;
+            return -1;
+        }
+        //
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:174]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 15]
```diff
@@ -126,14 +143,46 @@ int main(int argc, char **argv) {
...
+
         std::vector<std::string> devices{detector_mode, reid_mode};
-        slog::info << *InferenceEngine::GetInferenceEngineVersion() << slog::endl;
-        InferenceEngine::Core ie =
-            LoadInferenceEngine(devices, custom_cpu_library, path_to_custom_layers);
 
-        DetectorConfig detector_confid(det_model);
-        ObjectDetector pedestrian_detector(detector_confid, ie, detector_mode);
+        slog::info << *GetInferenceEngineVersion() << slog::endl;
+        //model API part
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:179]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 16]
```diff
@@ -126,14 +143,46 @@ int main(int argc, char **argv) {
...
+        InferenceEngine::Core ie;
 
+        auto execNet = detection_model->loadExecutableNetwork(
+            ConfigFactory::getUserConfig(FLAGS_d_det, FLAGS_l, FLAGS_c, FLAGS_nireq, FLAGS_nstreams, FLAGS_nthreads), ie);
+        auto req = std::make_shared<InferenceEngine::InferRequest>(execNet.CreateInferRequest());
+        //
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:185]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 17]
```diff
@@ -160,10 +209,72 @@ int main(int argc, char **argv) {
...
+                    cv::Point(static_cast<int>(round(static_cast<double>(x1))),
+                        static_cast<int>(round(static_cast<double>(y1)))));
+
+                if (object.rect.area() > 0) {
+                    detections.emplace_back(object);
+                }
+
+
+            }
+           //
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:277]:
```suggestion
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 18]
```diff
@@ -23,6 +23,17 @@
 #include <string>
 #include <gflags/gflags.h>
 
+//from model API
```
Comment (ivikhrev) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:26]:
```suggestion
```

**Comment by vladimir-dudnik** [comment]
pre-commit check failed
10:39:44  RuntimeError: demos/pedestrian_tracker_demo/cpp/README.md: "efficientdet-d0-tf" model reference is missing. Add it to README.md or update models.lst file.

File: demos/pedestrian_tracker_demo/cpp/src/tracker.cpp [hunk 19]
```diff
@@ -35,7 +35,7 @@ DetectionLog ConvertTracksToDetectionLog(const ObjectTracks& tracks) {
     DetectionLog log;
 
     // Combine detected objects by respective frame indices.
-    std::map<int, TrackedObjects> objects;
+    std::map<int64_t, TrackedObjects> objects;
```
Comment (vladimir-dudnik) [inline demos/pedestrian_tracker_demo/cpp/src/tracker.cpp:38]: why data type changed? What is the reason?

Comment (VoronovaIntern) [reply demos/pedestrian_tracker_demo/cpp/src/tracker.cpp:38]: @vladimir-dudnik the model API uses the int_64_t type for "frameId", then it is the first key in this structure

Comment (vladimir-dudnik) [reply demos/pedestrian_tracker_demo/cpp/src/tracker.cpp:38]: thanks, ok, then

File: demos/pedestrian_tracker_demo/cpp/CMakeLists.txt [hunk 20]
```diff
@@ -2,11 +2,14 @@
...
+
+
+
 
 add_demo(NAME pedestrian_tracker_demo
     SOURCES ${SOURCES}
     HEADERS ${HEADERS}
     INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/include"
-    DEPENDENCIES monitors)
+    DEPENDENCIES monitors models)
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/CMakeLists.txt:15]:
```suggestion
    DEPENDENCIES monitors models pipelines)
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 21]
```diff
@@ -23,6 +22,13 @@
 #include <string>
 #include <gflags/gflags.h>
 
+#include <models/detection_model_centernet.h>
+#include <models/detection_model_ssd.h>
+#include <models/detection_model_yolo.h>
+#include "../../common/cpp/pipelines/include/pipelines/metadata.h"
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:28]:
```suggestion
#include <pipelines/metadata.h>
```

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 22]
```diff
@@ -23,6 +22,13 @@
...
+
+using namespace InferenceEngine;
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:30]:
```suggestion
```
Please, add namespace directly where it is used, according to [the PR ](https://github.com/openvinotoolkit/open_model_zoo/pull/2661/files)

File: demos/pedestrian_tracker_demo/cpp/main.cpp [hunk 23]
```diff
@@ -224,8 +299,6 @@ int main(int argc, char **argv) {
                 PrintDetectionLog(log);
         }
 
-        slog::info << "Metrics report:" << slog::endl;
-        metrics.logTotal();
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/main.cpp:228]: Undo the deletion of lines introduced in develop branch (and lines 210-211, 191-192).

File: demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp [hunk 24]
```diff
@@ -40,6 +40,18 @@ static const char delay_message[] = "Optional. Delay between frames used for vis
...
+static const char input_resizable_message[] = "Optional. Enables resizable input with support of ROI crop & auto resize.";
+static const char iou_thresh_output_message[] = "Optional. Filtering intersection over union threshold for overlapping boxes.";
+static const char yolo_af_message[] = "Optional. Use advanced postprocessing/filtering algorithm for YOLO.";
+static const char labels_message[] = "Optional. Path to a file with labels mapping.";
+static const char nireq_message[] = "Optional. Number of infer requests. If this option is omitted, number of infer requests is determined automatically.";
+static const char num_threads_message[] = "Optional. Number of threads.";
+static const char num_streams_message[] = "Optional. Number of streams to use for inference on the CPU or/and GPU in "
+"throughput mode (for HETERO and MULTI device cases use format "
+"<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)";
+static const char person_label_message[] = "Optional. Label of class person for detector. Default -1 for tracking all objects";
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp:54]: The description of new parameters should be also added to README file (running section).

File: demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp [hunk 25]
```diff
@@ -40,6 +40,18 @@ static const char delay_message[] = "Optional. Delay between frames used for vis
 static const char output_log_message[] = "Optional. The file name to write output log file with results of pedestrian tracking. "
                                           "The format of the log file is compatible with MOTChallenge format.";
 static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+static const char at_message[] = "Required. Architecture type: centernet, ssd or yolo";
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp:43]:
```suggestion
static const char at_message[] = "Required. Architecture type for detector model: centernet, ssd or yolo";
```
The same for nireq, nstreams, nthreads parameters. In demos with two models it is not obvious, for which model these options can be set.

**Review by anzhella-pankratova — CHANGES_REQUESTED** [review]

File: demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp (hunk 25 above)
Comment (VoronovaIntern) [reply demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp]: done

File: demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp (hunk 24 above)
Comment (VoronovaIntern) [reply demos/pedestrian_tracker_demo/cpp/include/pedestrian_tracker_demo.hpp]: done

File: demos/pedestrian_tracker_demo/cpp/main.cpp (hunk 23 above)
Comment (VoronovaIntern) [reply demos/pedestrian_tracker_demo/cpp/main.cpp]: done

File: demos/pedestrian_tracker_demo/cpp/main.cpp (hunk 22 above)
Comment (VoronovaIntern) [reply demos/pedestrian_tracker_demo/cpp/main.cpp]: done

File: demos/pedestrian_tracker_demo/cpp/main.cpp (hunk 23 above)
Comment (anzhella-pankratova) [reply demos/pedestrian_tracker_demo/cpp/main.cpp]: `presenter.drawGraphs(frame);`
`metrics.update(startTime, frame, { 10, 22 }, cv::FONT_HERSHEY_COMPLEX, 0.65);`
these lines should be located after **drawing tracked detections** part, otherwise drawing time is not measured and included in performance metrics. Please, put it in a correct place.

File: demos/pedestrian_tracker_demo/cpp/README.md [hunk 26]
```diff
@@ -70,6 +116,7 @@ Options:
     -limit "<num>"               Optional. Number of frames to store in output. If 0 is set, all frames are stored.
     -m_det "<path>"              Required. Path to the Pedestrian Detection Retail model (.xml) file.
     -m_reid "<path>"             Required. Path to the Pedestrian Reidentification Retail model (.xml) file.
+	-at \"<type>\"               Required. Required. Architecture type for detector model: centernet, ssd or yolo.
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/README.md:119]:
```suggestion
	 -at \"<type>\"               Required. Architecture type for detector model: centernet, ssd or yolo.
```

File: demos/pedestrian_tracker_demo/cpp/README.md [hunk 27]
```diff
@@ -80,6 +127,15 @@ Options:
     -delay                       Optional. Delay between frames used for visualization. If negative, the visualization is turned off (like with the option 'no_show'). If zero, the visualization is made frame-by-frame.
     -out "<path>"                Optional. The file name to write output log file with results of pedestrian tracking. The format of the log file is compatible with MOTChallenge format.
     -u                           Optional. List of monitors to show initially.
+	-t                           Optional. Probability threshold for detections.
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/README.md:130]:
```suggestion
	 -t                           Optional. Probability threshold for detections.
```

File: demos/pedestrian_tracker_demo/cpp/README.md [hunk 28]
```diff
@@ -80,6 +127,15 @@ Options:
...
+    -auto_resize                 Optional. Enables resizable input with support of ROI crop & auto resize.
+    -iou_t                       Optional. Filtering intersection over union threshold for overlapping boxes.
+    -yolo_af                     Optional. Use advanced postprocessing/filtering algorithm for YOLO.
+    -labels \"<path>\"           Optional. Path to a file with labels mapping.
+    -nireq \"<integer>\"         Optional. Number of infer requests for detector model. If this option is omitted, number of infer requests is determined automatically.
+    -nstreams                    Optional. Number of streams to use for inference on the CPU or/and GPU in throughput mode for detector model(for HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)
```
Comment (anzhella-pankratova) [inline demos/pedestrian_tracker_demo/cpp/README.md:136]:
```suggestion
    -nstreams                    Optional. Number of streams to use for inference on the CPU or/and GPU in throughput mode for detector model (for HETERO and MULTI device cases use format <device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)
```

**Review by vladimir-dudnik — COMMENTED** [review]
@VoronovaIntern please add 2021 to copyrights for affected files

File: demos/pedestrian_tracker_demo/cpp/include/descriptor.hpp [hunk 29]
```diff
@@ -107,7 +107,7 @@ class DescriptorIE : public IImageDescriptor {
     VectorCNN handler;
 
 public:
-    DescriptorIE(const CnnConfig& config,
+    DescriptorIE(const CnnConfigTracker& config,
                  const InferenceEngine::Core& ie,
                  const std::string & deviceName):
```
Comment (vladimir-dudnik) [inline demos/pedestrian_tracker_demo/cpp/include/descriptor.hpp:112]:
```suggestion
                 const std::string& deviceName):
```

File: demos/pedestrian_tracker_demo/cpp/README.md [hunk 30]
```diff
@@ -70,6 +116,7 @@ Options:
     -limit "<num>"               Optional. Number of frames to store in output. If 0 is set, all frames are stored.
     -m_det "<path>"              Required. Path to the Pedestrian Detection Retail model (.xml) file.
     -m_reid "<path>"             Required. Path to the Pedestrian Reidentification Retail model (.xml) file.
+	 -at \"<type>\"               Required. Architecture type for detector model: centernet, ssd or yolo.
```
Comment (vladimir-dudnik) [inline demos/pedestrian_tracker_demo/cpp/README.md:119]:
```suggestion
    -m_det "<path>"              Required. Path to the Pedestrian Detection Retail model (.xml) file.
    -at \"<type>\"                  Required. Architecture type for detector model: centernet, ssd or yolo.
    -m_reid "<path>"             Required. Path to the Pedestrian Reidentification Retail model (.xml) file.
```

**Comment by VoronovaIntern** [comment]
@vladimir-dudnik please tell me how to do this "please add 2021 to copyrights for affected files "

**Comment by VoronovaIntern** [comment]
@vladimir-dudnik I've done it. Hope, that it's right

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/pedestrian_tracker_demo/cpp/main.cpp (hunk 5 above)
Comment (vladimir-dudnik) [reply demos/pedestrian_tracker_demo/cpp/main.cpp]: @VoronovaIntern please make comments if you have addressed questions from review, otherwise it is not clear if you agreed to make changes or not agreed, or just missed/ignored comment

Comment (VoronovaIntern) [reply demos/pedestrian_tracker_demo/cpp/main.cpp]: @ivikhrev @vladimir-dudnik  I've done it

File: demos/pedestrian_tracker_demo/cpp/main.cpp (hunk 13 above)
Comment (VoronovaIntern) [reply demos/pedestrian_tracker_demo/cpp/main.cpp]: I've done

File: demos/pedestrian_tracker_demo/cpp/README.md [hunk 31]
```diff
@@ -80,6 +127,15 @@ Options:
     -delay                       Optional. Delay between frames used for visualization. If negative, the visualization is turned off (like with the option 'no_show'). If zero, the visualization is made frame-by-frame.
     -out "<path>"                Optional. The file name to write output log file with results of pedestrian tracking. The format of the log file is compatible with MOTChallenge format.
     -u                           Optional. List of monitors to show initially.
+	 -t                           Optional. Probability threshold for detections.
```
Comment (vladimir-dudnik) [inline demos/pedestrian_tracker_demo/cpp/README.md:130]:
```suggestion
    -t                           Optional. Probability threshold for detections.
```

replaced TAB to SPACEs

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]
Please also add -at ssd to the sample command
```sh
./pedestrian_tracker_demo -i <path_video_file> \
                          -m_det <path_to_model>/person-detection-retail-0013.xml \
                          -m_reid <path_to_model>/person-reidentification-retail-0277.xml \
                          -d_det GPU
```

**Comment by VoronovaIntern** [comment]
@vladimir-dudnik before me

> Please also add -at ssd to the sample command
> 
> ```shell
> ./pedestrian_tracker_demo -i <path_video_file> \
>                           -m_det <path_to_model>/person-detection-retail-0013.xml \
>                           -m_reid <path_to_model>/person-reidentification-retail-0277.xml \
>                           -d_det GPU
> ```

done

**Review by vladimir-dudnik — APPROVED** [review]
